backend/src/data/migrations/20220830085215_add_tasks_table.ts

```diff
@@ -0,0 +1,50 @@
+import { Knex } from 'knex';
+
+enum TableName {
+  TASKS = 'tasks',
+  MENTEES_TO_MENTORS = 'mentees_to_mentors',
+  COURSE_MODULES = 'course_modules',
+}
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  MENTEES_TO_MENTORS_ID = 'mentees_to_mentors_id',
+  MODULE_ID = 'module_id',
+  STATUS = 'status',
+}
+
+const DELETE_STRATEGY = 'CASCADE';
+
+async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(TableName.TASKS, (table) => {
+    table.increments(ColumnName.ID).primary();
+    table
+      .dateTime(ColumnName.CREATED_AT)
+      .notNullable()
+      .defaultTo(knex.fn.now());
+    table
+      .dateTime(ColumnName.UPDATED_AT)
+      .notNullable()
+      .defaultTo(knex.fn.now());
+    table
```

Comment:
Should not on delete strategy be used here too?

Reply:
On our daily call, we were told, that if user changes a mentor, the progress is not deleted. I believe it should the same for delete